packages/grafana-ui/src/components/PanelChrome/PanelChrome.test.tsx

```diff
@@ -35,18 +37,50 @@ it('renders an empty panel with padding', () => {
   expect(screen.getByText("Panel's Content").parentElement).not.toHaveStyle({ padding: '0px' });
 });
 
-it('renders panel with a header if prop title', () => {
+// Check for backwards compatibility
```

Comment:
Is this backward compatibility? Do we plan to deprecate the title? I think we agreed on the title property should be present. So it is not backward compatibility if the API is still the same after the new panel chrome. wdyt?

Reply:
The `title` is an optional prop and as far as I know it will continue to be. We did mention something about making it mandatory for specific types of panels (hovering header was it?), but I don't think we made a decision to do so. 
The idea of the new API is to always rely on the prop `hoverHeader` to decide if we have a header or not. Regardless of the presence of any or all other props.

Reply:
> We did mention something about making it mandatory for specific types of panels (hovering header was it?)

No, sorry. It should be optional, you're right.

> The idea of the new API is to always rely on the prop hoverHeader to decide if we have a header or not. 

Ok, I understand, ignore my comment then :)